maths/area.py

```diff
@@ -458,6 +458,49 @@ def area_rhombus(diagonal_1: float, diagonal_2: float) -> float:
     return 1 / 2 * diagonal_1 * diagonal_2

```

Comment:
Perhaps add some doctests here too.

Reply:
Hey there! doctests were already present in there. I wonder if that's a mistake.